### ISSUE TITLE
Provide DataONE format list to checks

### ISSUE THREAD

**Issue by gothub** [NCEAS/metadig-engine#215]
A list of DataONE formats should be provided to checks, such as the 'NonProprietaryFormat' check so that the check doesn't have to read config info to determine where the check list file is located. This file is derived from the DataONE 'format' service and manually maintained in the same location as other engine info, i.e. /opt/local/metadig, but the file path is kept in the metadig config file and read by the engine.

**Comment by gothub** [comment]
The file DataONEformats.csv has been included with the metadig-checks repo and distribution starting with the metadig-checks commit b09556d35f8e9e30570e0606a280856b2d6bc619

**Comment by mbjones** [comment]
@gothub how do you ensure that this file gets updated when new formats are added? We are about to add some new formats (see [milestone 1.23](https://github.com/DataONEorg/object-formats/milestone/1)), and I expect the format list will change more frequently as we have opened up the process for adding formats to the broader DataONE community.

**Comment by gothub** [comment]
Yeah, I'm always hesitant to freeze data in local file, however, this list is curated as we denote every format that was manually determined to be proprietary. The most current issue for this is closed in the metadig-checks repo: https://github.com/NCEAS/metadig-checks/issues/23

As this file is curated, the current way to update it is with a new release of metadig-checks. This will be added to the release process. BTW - the check that uses this file considers a format non-proprietary if it is not found.
